Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -606,6 +607,11 @@ namespace AzToolsFramework
             }
         }
 
+        static bool IsFolderEmpty(AZStd::string_view path)
+        {
+            return QDir(path.data()).entryList(QDir::NoDotAndDotDot | QDir::AllEntries).count() == 0;
```

Comment:
Nit: Since the entryList is a `QStringList` you could just check `isEmpty()`

```suggestion
            return QDir(path.data()).entryList(QDir::NoDotAndDotDot | QDir::AllEntries).isEmpty();
```

Reply:
Changed now.